android/app/src/main/res/values/color.xml

```diff
@@ -22,4 +22,5 @@
     <color name="timestamp">#a0a3a7</color>
     <color name="link">#0a80d1</color>
     <color name="card_title">#ffffff</color>
+    <color name="snackbar_action_text">#dcb71c1c</color>
```

Comment:
This is hardcoded text and won't change if the developer wants to use theming option of app generator

Reply:
@iamareebjamal Sorry about that. I will fetch the accent colour dynamically and adjust opacity.